bootstrap/scss/_custom-styles.scss

```
@@ -83,6 +83,9 @@ footer {
     padding: 5rem; 
     background-color: #e9ecef;
     opacity: 0.7;
+    @include media-breakpoint-down(sm) {
+        padding: map_get($spacers, 3) + 0.25;
```

```suggestion
        padding: map_get($spacers, 3) * 1.1;
```

If we should choose higher values for spacing, then this padding will also grow.

---

default 1
```
$spacers: map-merge(
  (
    0: 0,
    1: ($spacer * .25),
    2: ($spacer * .5),
    3: $spacer,
    4: ($spacer * 1.5),
    5: ($spacer * 3)
  ),
  $spacers
);

```
we need 1.25, i made 1.25

---

it was easier to use what I did at the very beginning 1.25rem this is my opinion
For those who are not particularly familiar with bootstrap will need to see what these functions will output

---

I only wanted to point out, that the operator is not correct here. I have not calculated anything.

---

Understood thanks